README.md

```diff
@@ -50,6 +50,90 @@ python app.py
 
 5. Go to ` http://127.0.0.1:5000/` and enjoy the application.
 
+
+## Contribution Guidelines 🏗
+   
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further
+
+To start contributing, follow the below guidelines: 
+
+**🌟.**  Star🌟 the project to bookmark and appreciate the work.
+
+**0.**  Take a look at the existing [issues](https://github.com/sameersrivastava13/Vaccine-Tracker-App/issues) or create your own issues. Wait for the Issue to be assigned to you after which you can start working on it.
+
+**1.**  Fork [this](https://github.com/sameersrivastava13/Vaccine-Tracker-App) repository.
+
+**2.**  Clone your forked copy of the project.
+
+```
+git clone --depth 1 https://github.com/<your_user_name>/Vaccine-Tracker-App.git
+```
+
+**3.** Navigate to the project directory :file_folder: .
+
+```
+cd Vaccine-Tracker-App
+```
+
+**4.** Add a reference(remote) to the original repository.
+
+```
+git remote add upstream https://github.com/sameersrivastava13/Vaccine-Tracker-App.git
+
+```
+*5.** Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+```
+git pull upstream master
+```
+
+**7.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**8.** Perform your desired changes to the code base.
+
+<p align="center"><img width=35% src="https://media2.giphy.com/media/L1R1tvI9svkIWwpVYr/giphy.gif?cid=ecf05e47pzi2rpig0vc8pjusra8hiai1b91zgiywvbubu9vu&rid=giphy.gif"></p>
+
+**9.** Track your changes:heavy_check_mark: .
+
+```
+git add . 
+```
+
+**10.** Commit your changes .
+
+```
+git commit -m "Relevant message"
+```
+
+**11.** Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**12.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are supposed to make a PR to.
+
+**13.** Add an appropriate title and description to your pull request explaining your changes and efforts done.
+
+**14.** Click on `Create Pull Request`.
+
+**15.** Voila :exclamation: You have made a PR to the  sameersrivastava13/Vaccine-Tracker-App project :boom: . Sit back patiently and relax while the project maintainers review your PR. Please understand, at times the time taken to review a PR can vary from a few hours to a few days.
+
+<p align="center"><img src="https://media.tenor.com/images/b562ddcfb131e962f9dfa01bd32a30d1/tenor.gif" width=30%></p>
+
+## Project Admin 👨: sameersrivastava13
+
+
```

Comment:
@shivshankar9 you have repeated the same` contribution.md` file information in `readme.md` file. Please remove the entire part from the readme.md file. Rest everything looks fine to me.

Reply:
Ok